src/js/game/actions/CardActions.js

```diff
@@ -19,4 +20,19 @@ export class CardActions  {
     });
   }
 
+  dialectChelange(){
+    this.game.interface.lock().then(() => {
+      return this.game.popups.open({
+        type: 'confirmChallenge',
+        state: {
+          yesAction: 'dialectChelangePass',
```

Comment:
dialectChelangePass events is not implemented. Implement all events and finish task before pull request.